### PR TITLE
feat(daily): add reduced ringmaps to daily config

### PR DESCRIPTION
**Pull request by ljgray**
Somehow the previous PR wouldn't reflect updates to the branch, so it's easier to just make a new one since this is minor.

Goes with radiocosmology/draco#241

Updated to include both XX and YY pol

### CODE REVIEW

File: ch_pipeline/processing/daily.py [hunk 1]
```diff
@@ -622,7 +622,75 @@
...
+    - type: draco.analysis.transform.Reduce
+      in: ringmap_int_hpf
+      out: ringmap_reduced_freq
+      params:
+        axes: ["freq"]
+        datasets: ["map"]
+        op: "var"
+        selections: {"pol":[0, 3]}
+        generate_weight: true
+        apply_weight: false
```
Comment (sjforeman) [inline ch_pipeline/processing/daily.py:636]: Making sure params agree with `Reduce` task:
```suggestion
        generate_mask: true
        apply_mask: false
```

File: ch_pipeline/processing/daily.py [hunk 2]
```diff
@@ -622,7 +622,75 @@
...
+    - type: draco.analysis.transform.Reduce
+      in: ringmap_int_hpf
+      out: ringmap_reduced_el
+      params:
+          axes: ["el"]
+          datasets: ["map"]
+          op: "var"
+          selections: {"pol":[0, 3]}
+          generate_weight: true
+          apply_weight: false
```
Comment (sjforeman) [inline ch_pipeline/processing/daily.py:668]: Making sure params agree with `Reduce` task:
```suggestion
          generate_mask: true
          apply_mask: false
```

**Review by sjforeman — APPROVED** [review]
This looks good after fixing a few of the `Reduce` params

File: ch_pipeline/processing/daily.py (hunk 1 above)
Comment (ljgray) [reply ch_pipeline/processing/daily.py:636]: Good catch, thanks. I might actually have to change this again after reworking the `Reduce` task a bit

**Review by sjforeman — APPROVED** [review]
Looks good to merge after the corresponding draco PR

**Pull request merged**